spec.html

```diff
@@ -32156,6 +32156,33 @@ <h1>
         </emu-alg>
       </emu-clause>
 
+      <emu-clause id="sec-time-zone-identifiers">
+        <h1>Time Zone Identifiers</h1>
+
+        <p>
+          Time zones in ECMAScript are described using <dfn variants="time zone identifier">time zone identifiers</dfn>, which must be Strings composed entirely of characters from the Unicode Basic Latin block.
+          Time zone identifiers are case-insensitive and so are compared using ASCII case-insensitive comparisons.
+          Built-in time zones may be <dfn variants="named time zone">named time zones</dfn>, represented by the [[Identifier]] property of the records returned by AvailableNamedTimeZoneIdentifiers.
```

Comment:
```suggestion
          Built-in time zones may be <dfn variants="named time zone">named time zones</dfn>, represented by the [[Identifier]] property of the Record values returned by AvailableNamedTimeZoneIdentifiers.
```

Reply:
```suggestion
          Built-in time zones may be <dfn variants="named time zone">named time zones</dfn>, represented by the [[Identifier]] property of the Time Zone Identifier Records returned by AvailableNamedTimeZoneIdentifiers.
```

Reply:
Duelling suggestions! ⚔️  @michaelficarra's suggestion seems to encompass what @syg was getting at, so I'll push Michael's suggestion and assume that's OK. If you guys prefer Shu's suggestion better, let me know and I can switch.

Reply:
We should also probably call it a field, not a property.